conf/local.conf

```diff
@@ -1,19 +1,58 @@
 # Copyright (C) 2021 AVNET Embedded, MSC Technologies GmbH
 
-MACHINE = "sm2s-imx8mm"
+MACHINE = "sm2s-am6254"
 ACCEPT_FSL_EULA = "1"
-PACKAGE_CLASSES = "package_rpm"
-EXTRA_IMAGE_FEATURES ?= "debug-tweaks"
+
+CONF_VERSION = "1"
+
+# default build to ipk. rpm create issue.
+PACKAGE_CLASSES = "package_ipk"
+
+SDKMACHINE ?= "x86_64"
+
+# We default to enabling the debugging tweaks.
+EXTRA_IMAGE_FEATURES = "debug-tweaks"
```

Comment:
Do you need SDKMACHINE and EXTRA_IMAGE_FEATURES in any combination?

Reply:
Ok. will remove this and verified.
I have kept this as per recommended TI local.conf.

Reply:
One of the changes of debug-tweaks is that it removes the root password.
In msc-image-hwtests we don´t have root passwords anyway.
For the other images we have a documented password, so we should not remove it unless we want to change some documentation in the core.

Reply:
Understood. Thanks for info.